.github/workflows/ci.yml

```diff
@@ -29,14 +29,14 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        eslint: [6, 7, 8]
-        node: [12.22.0, 14, 16, 17, 18, 19, 20, 21]
+        eslint: [8]
+        node: [16, 17, 18, 19, 20, 21]
```

Comment:
this also dropped some node.js/eslint versions support. To reflect this in the generated changelog：
 1. update the PR title.
 2. put these changes to a separate PR.

Reply:
These changes are necessary to make CI pass. I'll officially change the Node.js supported versions in a separate PR.